core/keyboard_nav/navigation.js

```diff
@@ -94,10 +94,10 @@ Blockly.navigation.actionNames = {
   TOOLBOX: 'toolbox',
   EXIT: 'exit',
   TOGGLE_KEYBOARD_NAV: 'toggle_keyboard_nav',
-  MOVE_WS_CURSOR_UP: 'move_ws_cursor_up',
-  MOVE_WS_CURSOR_DOWN: 'move_ws_cursor_down',
-  MOVE_WS_CURSOR_LEFT: 'move_ws_cursor_left',
-  MOVE_WS_CURSOR_RIGHT: 'move_ws_cursor_right'
+  MOVE_WS_CURSOR_UP: 'move workspace cursor up',
```

Comment:
Curious why remove the underscore?

Reply:
Because the demo displays these action names to the user in the key map list.